### DISCUSSION TITLE
As a developer, I need to be able to contact my sponsors in some cases

### DISCUSSION THREAD

**Discussion by donmccurdy** [community/community#53072] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Product Feedback

### Body

As a developer, there are several cases in which it's critical that I be able to communicate with my sponsors, if only as a one-time outreach. Examples:

1. To give them my email address, so they can reach out for the perks of their sponsorship tier
2. To let them know if there's a problem, e.g. I cannot accept their sponsorship for some reason (e.g. religious or ethical concerns)
3. To alert them if there's a major change in my work, e.g. I am no longer maintaining the projects they care about

Because "Receive email updates from [account]" is *off* by default, sponsors have to explicitly opt in to receiving communication from the account they are sponsoring. I have no insight into how many sponsors do this. I have no interest in spamming my sponsors with newsletters, but I do need to communicate certain things if I'm going to maintain a positive relationship.

The "Display welcome message" box in the sponsorship tiers setup is a good feature, but I have no insight into how it works – if the sponsor doesn't sign up for my email updates, do they receive the welcome message?

Similarly, I have a single sponsor paying a larger amount in a certain tier. If I'm going to spend the time writing up a nice email to that tier, I would like to know that my email is going somewhere, and not disappearing into the void.

tl;dr — I am finding it difficult to maintain an appropriate level of transparency with my sponsors.

**Comment by anjuan** [comment, marked as answer]
Hi, @donmccurdy, I see that you're looking for more transparency when you reach out to your sponsors, specifically who receives your messages. You're correct, sponsors have to [opt in to receive updates about your work](https://docs.github.com/en/sponsors/receiving-sponsorships-through-github-sponsors/contacting-your-sponsors). Would it help to surface this information by doing something like showing which of your sponsors have opted in to receive emails?

Also, to answer your other point:

> if the sponsor doesn’t sign up for my email updates, do they receive the welcome message?

Yes, they receive the welcome message no matter what as part of the email they get to confirm their new sponsorship. You can share your contact email address if your welcome message. They can also see the welcome message at any time when viewing your Sponsors profile

**Reply by donmccurdy** [reply]
Thanks @anjuan! Yes, I think any further information (number of sponsors, number of sponsors per tier, and/or the list of sponsors) would be really helpful for me to know if/when I can use the newsletter feature.

> Yes, they receive the welcome message no matter what as part of the email they get to confirm their new sponsorship... They can also see the welcome message at any time when viewing your Sponsors profile.

OK, I'd been including a contact email address in the welcome message but wasn't sure if all sponsors saw it. That's very helpful to know.

Thanks for your response!
